android/filament-utils-android/CMakeLists.txt

```diff
@@ -38,6 +39,12 @@ add_library(filament-utils-jni SHARED
         ../common/NioUtils.cpp
 )
 
+target_include_directories(filament-utils-jni PRIVATE
+        ${FILAMENT_DIR}/include
+        ..
+        ../../third_party/stb
```

Comment:
Any chance we could reuse our own HDR decoder from libimage? Even if we compile it separately. I'm asking because there are some subtle differences between HDR decoders and it would be good to match cmgen.

Reply:
Good point I'll give it a try.  It would give us EXR reading capability too.

Reply:
Well I'm wondering if we want to link the entire libimage, it's not small (and supports PNG that we don't care about here). We could however compile in only our own HDRDecoder class.